test/Concurrency/issue-70019.swift

```diff
@@ -0,0 +1,17 @@
+// RUN: %target-typecheck-verify-swift -swift-version 5
+// RUN: %target-typecheck-verify-swift -swift-version 5 -enable-library-evolution -I %t
```

Comment:
What’s `-I %t` for?

Reply:
Let's ask @xedin.

Reply:
In this case it's not needed, I copied it from `split-file` test without realizing it.